docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -2950,28 +2953,40 @@ Create a volume
 
     {
       "Name": "tardis",
-      "Driver": "local",
+      "Driver": "custom",
       "Mountpoint": "/var/lib/docker/volumes/tardis",
-      "Status": null,
+      "Status": {
+        "hello": "world"
+      },
       "Labels": {
         "com.example.some-label": "some-value",
         "com.example.some-other-label": "some-other-value"
       },
+      "Scope": "local"
     }
 
 **Status codes**:
 
 - **201** - no error
 - **500**  - server error
 
-**JSON parameters**:
+**JSON parameters in request**:
 
 - **Name** - The new volume's name. If not specified, Docker generates a name.
 - **Driver** - Name of the volume driver to use. Defaults to `local` for the name.
 - **DriverOpts** - A mapping of driver options and values. These options are
     passed directly to the driver and are driver specific.
 - **Labels** - Labels to set on the volume, specified as a map: `{"key":"value" [,"key2":"value2"]}`
 
+**JSON parameters in response**:
```

Comment:
Also to save ourself some work maintaining these descriptions, I think we can link to the "inspect" section;

```Markdown
**JSON fields in response**:

Refer to the [Inspect a volume](#inspect-a-volume) section or details about the
JSON fields returned in the response.
```